MAUI/DigitalGauge/migration.md

```diff
@@ -0,0 +1,367 @@
+</tr>
+</table>
+
+The following code example explains how to initialize the character type in Xamarin SfDigitalGauge and .NET MAUI SfDigitalGauge.
+
+<table>
+<tr>
+<th>Xamarin SfDigitalGauge</th>
+<th>.NET MAUI SfDigitalGauge</th></tr>
+<tr>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<gauge:SfDigitalGauge Value="SYNCFUSION" SegmentStrokeWidth="4" 
+                          HeightRequest="100"  WidthRequest="340"
+                          DisabledSegmentAlpha="25" DisabledSegmentColor="Gray"
+                          CharacterHeight="90" CharacterWidth="25" 
+                          HorizontalOptions="Center" VerticalOptions="Center"
+                          CharacterType="SegmentSeven" 
+                         CharacterStrokeColor="#146CED"/>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfDigitalGauge digital = new SfDigitalGauge();
+
+            digital.HeightRequest = 100;
+
+            digital.WidthRequest = 340;
+
+            digital.Value = "SYNCFUSION";
+
+            digital.CharacterHeight = 90;
+
+            digital.CharacterWidth = 25;
+
+            digital.HorizontalOptions = LayoutOptions.Center;
+
+            digital.VerticalOptions = LayoutOptions.Center;
+
+            digital.SegmentStrokeWidth = 4;
+
+            digital.CharacterType = CharacterType.SegmentSeven;
+
+            digital.DisabledSegmentAlpha = 25;
+
+            digital.CharacterStrokeColor = Color.FromRgb(20, 108, 237);
+
+            digital.DisabledSegmentColor = Color.Gray;
+
+
+{% endhighlight %}
+
+{% endtabs %}
+</td>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<gauge:SfDigitalGauge Text="1 2 3 4 5" StrokeWidth="5" 
+                          HeightRequest="100"  WidthRequest="300"
+                          DisabledSegmentAlpha="25" DisabledSegmentStroke="LightSkyBlue"
+                          CharacterHeight="90" CharacterWidth="25" 
+                          HorizontalOptions="Center" VerticalOptions="Center"
+                          CharacterType="SevenSegment" 
+                         CharacterStroke="#146CED"/>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfDigitalGauge digital = new SfDigitalGauge();
+
+            digital.HeightRequest = 100;
+
+            digital.WidthRequest = 300;
+
+            this.BackgroundColor = Color.White;
+
+            digital.Text = "1 2 3 4 5";
+
+            digital.CharacterHeight = 90;
+
+            digital.CharacterWidth = 25;
+
+            digital.HorizontalOptions = LayoutOptions.Center;
+
+            digital.VerticalOptions = LayoutOptions.Center;
+
+            digital.StrokeWidth = 5;
+
+            digital.CharacterType = CharacterType.SevenSegment;
+
+            digital.DisabledSegmentAlpha = 25;
+
+            digital.CharacterStroke = Color.FromRgb(20, 108, 237);
+
+            digital.DisabledSegmentStroke = Color.LightSkyBlue;
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
```

Comment:
after the add Topic

## Event

https://help.syncfusion.com/cr/xamarin/Syncfusion.SfGauge.XForms.SfDigitalGauge.html#Syncfusion_SfGauge_XForms_SfDigitalGauge_ValueChanged